packages/helpers/classes/request.d.ts

```diff
@@ -7,4 +9,5 @@ export default interface RequestOptions<TData = any, TParams = object> {
     qs?: TParams;
     body?: TData;
     headers?: object;
+    httpsAgent?: https.Agent;
```

Comment:
I believe this data is stored in `headers`.

Reply:
this is a top-level field on an `axios` request. i don't think we'd want to pass it in headers b/c it won't actually get sent across the wire, it's just connection settings essentially.

if we were to add it to headers, then there would need to be a code change in the JS file to filter it out and pass it as a top-level field in the axios request